stories/recipes/ConfirmationDialog.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import {
+  useDialog,
+  DialogCentreScreen,
+  ContentProvider,
+  Button,
+  ButtonPrimary,
+  P
+} from "../../src";
+
+export default {
+  title: "Recipes/Confirmation Dialog"
+};
+
+const contentNodes = {
+  "demo.ToggleButton": {
+    copy: {
+      en: "Toggle Dialog",
+      fr: "Toggle Dialog"
+    }
+  },
+  "demo.confirmationP": {
+    copy: {
+      en: "Are you sure?",
+      fr: "Êtes-vous sûr?"
+    }
+  },
+  "demo.yesButton": {
+    copy: {
+      en: "Yes",
+      fr: "Oui"
+    }
+  },
+  "demo.noButton": {
+    copy: {
+      en: "No",
+      fr: "Non"
+    }
+  }
+};
+
+const theme = {
+  colors: {
+    primary: {
+      light: "#d0dbfd",
+      dark: "#230cc2"
+    }
+  }
+};
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+`;
+
+const content = {
+  contentNodes,
+  theme
+};
+
+export const ConfirmationDialogDemo = () => {
+  const [confirmed, setConfirmed] = useState(false);
+
+  const handleConfirmation = userHasConfirmed => {
+    setConfirmed(userHasConfirmed);
+  };
+
+  return (
+    <ContentProvider lang="en" content={content}>
+      <ConfirmationDialog handleConfirmation={handleConfirmation}>
+        <P contentKey="demo.confirmationP" />
+      </ConfirmationDialog>
+      <div>confirmed: {`${confirmed}`}</div>
+    </ContentProvider>
+  );
+};
+
+const ConfirmationDialog = ({ handleConfirmation, children }) => {
+  const dialog = useDialog();
+  const YES = "yes";
+  const handleClick = e => {
+    e.preventDefault();
```

Comment:
nit: Not needed

Reply:
You're right it wouldn't make any difference afaik. I generally err on the side of preventing the default behaviour just to be safe. But please let us know if there are any disadvantages of doing so.